bin/test-lxd-network

```diff
@@ -221,6 +221,21 @@ sleep 10
 lxc list
 networkTests
 
+# Check bridged NIC type works with "agent.rename_interfaces".
+echo "=> Performing bridged NIC agent.rename_interfaces tests"
+lxc stop v1-physical
+lxc config set v1-physical agent.rename_interfaces=true
+lxc start v1-physical
+
+# Wait for lxd-agent to rename the interface.
+sleep 10
```

Comment:
This feels too short and at best will likely only work intermittently.
Can you use waitLXDAgent from https://github.com/lxc/lxc-ci/blob/master/bin/test-lxd-storage-disks-vm#L55 (copy it here)

Reply:
@tomponline I've done this, but in doing so I've noticed there are a lot of these small functions duplicated across the scripts. Is there a reason we're doing this instead of just sourcing/importing them?

Reply:
I thought about this originally, and I'm not dead set against it. But I've found that when running these scripts locally to test they work before submitting the PR some of them require to be run on special hardware (thinking SR-IOV and GPU ones for example), so being able to just copy a single file to a specific machine (and quickly iterating) to run it was easier. So laziness basically :)

Reply:
Hmm... Yeah I guess it's not really a problem unless we find an issue with one of the functions and have to change them all and I guess we can make a decision about it at that point.